rosidl_generator_py/resource/_msg_support.c.em

```diff
@@ -52,7 +54,22 @@ if not field.type.is_primitive_type() and field.type.is_array:
 @{
 msg_typename = '%s__%s__%s' % (spec.base_type.pkg_name, subfolder, spec.base_type.type)
 }@
+@
+@[for field in spec.fields]@
+@{lowercase_field_type = convert_camel_case_to_lower_case_underscore(field.type.type)}@
+@[  if not field.type.is_primitive_type()]@
+@[    if spec.base_type.pkg_name != field.type.pkg_name]@
+ROSIDL_GENERATOR_C_IMPORT
+@[    end if]@
+bool @(field.type.pkg_name)_@(lowercase_field_type)__convert_from_py(PyObject * _pymsg, void * _ros_message);
```

Comment:
I think this function should use a double underscore between the package name and the field type in order to clearly separate the two from each other (same as between the field type and the fixed name).

Same for `convert_to_py`.

Reply:
resolved in https://github.com/ros2/rosidl_python/commit/18d2af12adc34c322bf2e8ec8d49e66e62460e19